packages/ublue-os-media-automount-udev/ublue-os-media-automount-udev.spec

```
@@ -23,7 +23,7 @@ Supplements:    systemd-udev
 
 %install
 install -p -Dm0644 ./60-media-automount.rules %{buildroot}%{_exec_prefix}/lib/udev/rules.d/60-media-automount.rules
-install -p -Dm0644 ./is_in_fstab.sh %{buildroot}%{_libexecdir}/is_in_fstab.sh
+install -p -Dm0744 ./is_in_fstab.sh %{buildroot}%{_libexecdir}/is_in_fstab.sh
```

```suggestion
install -p -Dm0755 ./is_in_fstab.sh %{buildroot}%{_libexecdir}/is_in_fstab.sh
```
You probably want 755 since its an executable file on `/usr/libexec`. Thats what they usually do tbh